hooks/tk-multi-publish2/nuke/publish.py

```diff
@@ -548,17 +560,19 @@ def _write_node_path_duplicacy(self, item):
         if node_path in all_paths:
             duplicate_path_node = [key for (key, value) in item.parent.properties['write_node_paths_dict'].items()
                                    if value == node_path]
```

Comment:
If the fetch pattern is by node path, maybe the dict should be organised as `{"path1": ["node1", "node2"]}`? Then we can just ask for `dict[node_path]`
If it is also being accessed by node_name, feel free to ignore this.